README.md

```diff
@@ -1,2 +1,32 @@
-# nix
-Nix flakes, modules, and reused utility code across the InferenceQL ecosystem
+# InferenceQL/nix
+
+This repo holds Nix flake, modules, packages, and reusable utility Nix language code for use across the InferenceQL ecosystem.
+
+## Usage
+
+### Build an artifact
+
+Currently, you can build a package directly like so:
+
+```bash
+nix build github.com:InferenceQL/nix#ociImgBase
+```
+
+### Import utility code
+
+To access the `lib` code exported by this flake, declare this repo as a flake input:
+
+```nix
+{
+  inputs = {
+    nixpkgs.url = ...
+    iqlnix.url = "github"
```

Comment:
this would need the full GitHub path, correct?

Reply:
good catch. i must have stopped typing and got distracted.